.gas-snapshot

```diff
@@ -1,6 +1,201 @@
 BundleRegistryGasUsageTest:testGasRegister() (gas: 1630351)
 BundleRegistryGasUsageTest:testGasTrustedRegister() (gas: 1468291)
+BundleRegistryTest:testFuzzCannotChangeTrustedCallerToZeroAddress(address) (runs: 512, μ: 17362, ~: 17362)
```

Comment:
I run snapshot on a small selection of tests that model typical gas consumption and ignore the rest of the tests since they tend to change everytime you change the test logic (which is often) 

Running `forge snapshot --match-contract Gas` should resolve this issue

Reply:
Aha, that makes sense. Fixed this up.